source/uwp/NuGet/AdaptiveCards.Rendering.Uwp.nuspec

```diff
@@ -12,6 +12,10 @@
         <iconUrl>https://adaptivecards.io/content/icons_blue/blue-48.png</iconUrl>
         <license type="file">EULA-Windows.txt</license>
         <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+        <dependencies>
```

Comment:
There should be a major version change on Version somewhere. Possible version 3.1.0

Reply:
https://www.nuget.org/packages/AdaptiveCards.Rendering.Uwp/

Reply:
Makes sense. @beervoley any idea where that is?

Reply:
https://github.com/microsoft/AdaptiveCards/blob/main/custom.props
but it will first be updated in the release branch and then merged to main
looks like we forgot to update it last time